WebRoot/js/zimbraAdmin/servers/view/ZaNewVolumeXWizard.js

```diff
@@ -0,0 +1,1062 @@
+        numCols: 1,
+        items: [
+            {
+                type: _ZAWIZGROUP_,
+                isTabGroup: true,
+                numCols: 2,
+                colSizes: ["200px", "*"],
+                items: [
+                    {
+                        ref: ZaServer.A_VolumeName,
+                        type: _TEXTFIELD_,
+                        label: ZaMsg.LBL_VM_VolumeName,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "text-align:left;",
+                        width: 150,
+                    },
+                    {
+                        ref: ZaServer.A_VolumeRootPath,
+                        type: _TEXTFIELD_,
+                        label: ZaMsg.LBL_VM_VolumeRootPath,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "text-align:left;",
+                        width: 150,
+                    },
+                    {
+                        ref: ZaServer.A_VolumeType,
+                        type: _OSELECT1_,
+                        choices: ZaServer.volumeTypeChoices,
+                        width: 155,
+                        label: ZaMsg.LBL_VM_VolumeType,
+                        labelCssStyle: "text-align:left;",
+                    },
+                    {
+                        ref: ZaServer.A_VolumeCompressBlobs,
+                        type: _WIZ_CHECKBOX_,
+                        label: ZaMsg.VM_VolumeCompressBlobs,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "padding-left:3px;",
+                        trueValue: true,
+                        falseValue: false,
+                        align: _LEFT_,
+                    },
+                    {
+                        type: _GROUP_,
+                        numCols: 3,
+                        colSpan: 2,
+                        colSizes: ["200px", "*", "*"],
+                        items: [
+                            {
+                                ref: ZaServer.A_VolumeCompressionThreshold,
+                                type: _TEXTFIELD_,
+                                label: ZaMsg.LBL_VM_VolumeCompressThreshold,
+                                labelCssStyle: "text-align:left;",
+                                width: 100,
+                                labelLocation: _LEFT_,
+                            },
+                            {
+                                type: _OUTPUT_,
+                                label: null,
+                                labelLocation: _NONE_,
+                                value: ZaMsg.NAD_bytes,
+                                align: _LEFT_,
+                            },
+                        ],
+                    },
+                ],
+            },
+        ],
+    });
+
+    // New S3 volume step
+    ZaNewVolumeXWizard.NEW_S3_VOLUME = ZaNewVolumeXWizard.stepObject[ZaServer.A_S3_StoreProvider] = ++this.TAB_INDEX;
```

Comment:
just to confirm, is it correct?

Reply:
yes, this is done to match the storeProvider to its corresponding step in the form (used for navigation in createS3BucketCallback)